include/oc_uuid.h

```diff
@@ -141,6 +142,18 @@ void oc_gen_uuid(oc_uuid_t *uuid) OC_NONNULL();
 OC_API
 bool oc_uuid_is_equal(oc_uuid_t first, oc_uuid_t second);
 
+#ifdef OC_HAS_FEATURE_BRIDGE
```

Comment:
I wouldn't ifdef this, it's a simple function that will likely get optimized out, so lets add it globally

Reply:
done